### DISCUSSION TITLE
Data Badge

### DISCUSSION THREAD

**Discussion by jeffreylauwers** [nl-design-system/backlog#307] [Component suggestions]
<img width="480" alt="visuele weergave van de data badge component" src="https://github.com/user-attachments/assets/721e5309-2deb-45eb-b030-b17302481542">

We willen graag meer leren over de Data badge component. In welke vorm, of vormen, wordt deze momenteel ingezet en toegepast binnen jouw organisatie?

Deel hieronder een link naar Figma, Storybook, Github of screenshots. Met deze informatie kunnen we bewijzen dat de component onderdeel moet worden van de NL Design System component library.

Daarnaast wordt het delen van onderzoek gewaardeerd. Dus heb je zelf onderzoek gedaan en zijn er inzichten voor dit component? Dan horen we dit ook graag!

[Link naar bericht in Slack](https://codefornl.slack.com/archives/C7KDQRXMZ/p1726052401913379)

---

## Naam
Data Badge

## Doel
Label met extra informatie, zoals een categorie of een eigenschap.

## Gerelateerde componenten
[Dot Badge](https://github.com/orgs/nl-design-system/discussions/308), [Number Badge](https://github.com/orgs/nl-design-system/discussions/214), [Status Badge](https://github.com/orgs/nl-design-system/discussions/290)

## Richtlijnen
Acceptatiecriteria: [NL Design System Acceptatiecriteria voor Data Badge](https://nldesignsystem.nl/data-badge)

## Onderzoek
[Rapport Generieke Services 2023 - Gemeente Den Haag (PDF, 9 MB)](https://github.com/user-attachments/files/16993296/Rapport.generieke.services.23032023.def.pdf)
Uit dit gebruikersonderzoek blijkt dat de Data Badge heel goed werkt. Het wordt makkelijker begrepen dan het gebruik van enkel iconen. 

[Onderzoek Generieke Services - Miro](https://miro.com/app/board/uXjVP0OE2fg=/?share_link_id=511269957959)
Miro board dat is gebruikt tijdens het gebruikersonderzoek waarin inzichten te vinden zijn over de Data Badge. 

[Interaction Design Foundation - Preattentive Visual Properties](https://www.interaction-design.org/literature/article/preattentive-visual-properties-and-how-to-use-them-in-information-visualization#:~:text=Preattentive%20processing%20takes%20place%20in,than%20500%20milliseconds%20to%20complete.)
Artikel dat beschrijft dat door preattentieve verwerking bepaalde visuele eigenschappen zoals kleur, vorm en grootte snel worden waargenomen, wat belangrijk is voor de effectiviteit van Badges.

**Comment by Rozerinay** [comment]
## Amsterdam

- [Storybook](https://designsystem.amsterdam/?path=/docs/components-feedback-badge--docs)
- [Figma](https://www.figma.com/design/9IGm6IdPUYizBNGsUnueBd?node-id=17750-42420)
- [GitHub](https://github.com/Amsterdam/design-system/tree/develop/packages/css/src/components/badge)

---

### Screenshots / Varianten

**Storybook**
<img width="88" alt="image" src="https://github.com/user-attachments/assets/4e7ec715-b5ce-4560-ac1e-4f1e5d32a66d">


**Figma**
<img width="455" alt="image" src="https://github.com/user-attachments/assets/19030652-29a1-4088-9b89-0d8d9a9d2374">

---

### Notities

- Onder de naam: Badge.
- Beschikbaar in 14 kleuren. 
- Label kan een `string` of `number` zijn.
- Er worden nog richtlijnen bedacht omtrent het kleurgebruik.
- Documentatie beschikbaar (Figma).

**Comment by Rozerinay** [comment]
## DSO Toolkit

- [Documentatie website](https://www.dso-toolkit.nl/62.8.4/components/signals/label)
- [Storybook](https://storybook.dso-toolkit.nl/62.8.4/?path=/docs/html-css-badge--docs)
- [GitHub](https://github.com/dso-toolkit/dso-toolkit/tree/master/packages/dso-toolkit/src/components/badge)

---

### Screenshots / Varianten

**Storybook**
Default
<img width="100" alt="image" src="https://github.com/user-attachments/assets/74f1bb8c-419e-47a9-b41d-7c2cf9279ead">

Truncate
<img width="200" alt="image" src="https://github.com/user-attachments/assets/17f4e5ab-332e-46a2-a60f-ab181e5f0e7c">

With action
<img width="100" alt="image" src="https://github.com/user-attachments/assets/ad61c47c-8efb-41f6-a4a4-653934c42f96">

With symbol color
<img width="100" alt="image" src="https://github.com/user-attachments/assets/6f06213e-acda-4f5b-8917-b0f342410fe8">

With symbol image
<img width="100" alt="image" src="https://github.com/user-attachments/assets/59aa5cb5-94da-4a58-975d-02de0ce4045d">

---

### Notities

- Onder de naam: Label.
- Kan verschillende statussen bevatten, namelijk: primary, succes, info, warning, danger, bright, error en attention. 
- Niet zichtbaar of dit ook kan worden ingezet met 'data', zoals een datum.
- Kan een icoon of symbool bevatten. 
- Documentatie beschikbaar (documentatiewebsite).

**Comment by Rozerinay** [comment]
## Rotterdam

- [Storybook](https://nl-design-system.github.io/rotterdam/?path=/docs/css-utrecht-data-badge--docs)
- [Figma](https://www.figma.com/design/ZWSC4gCrOXRUR9UX3aoZ8x/Gemeente-Rotterdam-Design-System?node-id=6359-9671&node-type=frame&t=TNfidHbaqtx4DX4C-11)

---

### Screenshots / Varianten

**Storybook**

Default
<img width="100" alt="image" src="https://github.com/user-attachments/assets/4ed80bad-f8dc-4b56-a0a4-2b3fd1e71866">

Badge as link
<img width="100" alt="image" src="https://github.com/user-attachments/assets/e906de16-0cff-4264-87ec-a6b72b9b158f">

Light blue
<img width="100" alt="image" src="https://github.com/user-attachments/assets/c28b82e0-b618-49af-ad14-1c5fd6623484">

Blue
<img width="107" alt="image" src="https://github.com/user-attachments/assets/4a4f7419-e01d-4c72-a240-9c1245b496f6">

Dark blue
<img width="100" alt="image" src="https://github.com/user-attachments/assets/0578e4ee-4936-46ef-9445-b433982f827e">

Green blue
<img width="100" alt="image" src="https://github.com/user-attachments/assets/a753f480-c977-4b07-aec0-d7ad552ac737">

Red
<img width="100" alt="image" src="https://github.com/user-attachments/assets/603c8121-5c61-4b4e-912b-57c7d5c3a3e0">

Dark yellow
<img width="100" alt="image" src="https://github.com/user-attachments/assets/eaf49f9a-65e1-43ba-bb33-417a29bc8e1b">

---

### Notities

- Onder de naam: 'Data badge' in Storybook en 'Badge' in Figma.
- Niet zichtbaar of dit ook kan worden ingezet met 'data', zoals een datum. Label kan in ieder geval alleen een `string` zijn.
- Beschikbaar in 8 kleuren.
- Mogelijkheid om de badge aan te kunnen klikken (Badge as link).

**Comment by Rozerinay** [comment]
## Utrecht

- [Storybook](https://nl-design-system.github.io/utrecht/storybook/?path=/docs/css_css-data-badge--docs)
- [GitHub](https://github.com/nl-design-system/utrecht/tree/main/components/badge-data)

---

### Screenshots / Varianten

**Storybook**

Default
<img width="100" alt="image" src="https://github.com/user-attachments/assets/dc68290b-4119-4e12-b5f7-f81d88953a2b">

Machine-readable value
<img width="100" alt="image" src="https://github.com/user-attachments/assets/180a944b-e9d8-4993-829c-035505c569a9">

Machine-readable date
<img width="200" alt="image" src="https://github.com/user-attachments/assets/9a602a06-d68c-471f-a2e1-b292730dc415">

Machine-readable date and time
<img width="83" alt="image" src="https://github.com/user-attachments/assets/c6e1d509-840c-4814-97cb-3c8765ae78e6">

Machine-readable duration
<img width="207" alt="image" src="https://github.com/user-attachments/assets/e2270d9c-1e56-4421-93d1-a397f29da909">

---

### Notities

- Naam wordt nog gewijzigd van Badge data naar Data badge.
- Beschikbaar in 1 kleur. 
- Weergeeft onder andere tijdsgegevens en datums.
- Technische documentatie beschikbaar (Storybook).

**Comment by Rozerinay** [comment]
## DUO

[Documentatie website](https://uno.dfront.rijkscloud.nl/#/components/badge)

---

### Screenshots / Varianten

**Documentatie website**

Met achtergrond
Gebruik een badge met achtergrondkleur als je een specifieke status, een label of een bericht wilt communiceren. De achtergrondkleur van de badge trekt de aandacht en kan helpen om belangrijke informatie op te laten vallen. Bijvoorbeeld, een groene badge met de tekst "Goedgekeurd".

<img width="400" alt="image" src="https://github.com/user-attachments/assets/591edf1e-28b1-4d82-8faf-2ca251ed9f0f">

Omlijnd
Gebruik een badge zonder achtergrondkleur als de content minder belangrijk is voor de actie of informatiebehoefte van de gebruiker, bijvoorbeeld bij rentepercentages.

<img width="400" alt="image" src="https://github.com/user-attachments/assets/060364d9-1d1c-440b-abb5-9ed9b2402ee0">

Transparant
Gebruik een transparante grote badge voor datums en bestandsnamen.

<img width="100" alt="image" src="https://github.com/user-attachments/assets/8a6ca5a8-ec7c-4bb0-9356-5a7f0e4e0611">

---

### Notities

- Onder de naam: Badge.
- Beschikbaar in verschillende varianten.
- Zowel de variant die de status weergeeft als de variant die data presenteert, zijn samengebracht in 1 Badge component.
- Documentatie beschikbaar (documentatie website).

**Comment by Rozerinay** [comment]
## ✨ Dit component is nu Help Wanted ✨

Help je mee hem door de Community stappen te krijgen?
[Data Badge op het Community bord](https://github.com/orgs/nl-design-system/projects/29/views/1?filterQuery=data+badge)

**Comment by jeffreylauwers** [comment]
## ✨ Dit component is nu Community ✨

Help je mee hem door de Candidate stappen te krijgen?
[Data Badge op het Candidate bord](https://github.com/orgs/nl-design-system/projects/32/views/1?filterQuery=Data+Badge)

**Comment by jeffreylauwers** [comment]
## ✨ Dit component is nu Candidate ✨

Help je mee hem door de Hall of Fame stappen te krijgen?
[Color Sample op het Hall of Fame bord](https://github.com/orgs/nl-design-system/projects/30/views/1?filterQuery=data+badge)
